### PR TITLE
1639 dynamic wrapped client

### PR DESCRIPTION
**Pull request by mike-sirs**
# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) if any changes are user facing, including updates to minimum requirements e.g. Kubernetes version bumps
- [ ] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [ ] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [ ] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including functionality, content, code)
- [ ] Has a kind label. You can add one by adding a comment on this PR that contains `/kind <type>`. Valid types are bug, cleanup, design, documentation, feature, flake, misc, question, tep
- [ ] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings). See some examples of [good release notes](https://github.com/tektoncd/community/blob/main/standards.md#good-release-notes).
- [ ] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

release-note:
- go 1.21
- [remove go.mod versions fix](https://github.com/tektoncd/pipeline/issues/7187) 
- bump knative.dev
- dependencies update

### CODE REVIEW

**Comment by mike-sirs** [comment]
@khrm

**Review by khrm — COMMENTED** [review]
/ok-to-test

File: go.mod [hunk 1]
```diff
@@ -1,156 +1,142 @@
 module github.com/tektoncd/triggers
 
-go 1.19
+go 1.21
```
Comment (khrm) [inline go.mod:3]: Please don't change go version.

**Comment by savitaashture** [comment]
@mike-sirs can you rebase the PR ?

**Comment by mike-sirs** [comment]
Will do later today

File: go.mod (hunk 1 above)
Comment (mike-sirs) [reply go.mod:3]: OK, PR closed

**Comment by khrm** [comment]
@mike-sirs Why did you close this?

**Pull request closed without merging**